std_msgs/Header header
int32 rang
float32 x1
float32 y1
float32 x2
float32 y2
float32 x_centre
float32 y_centre
bool jouee
sensor_msgs/Image image

